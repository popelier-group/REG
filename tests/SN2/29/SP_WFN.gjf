# b3lyp/6-311++g(d,p) output=wfn

Single Point Energy

-1 1
 C                 -1.45468400   -0.00005300    0.00014200
 H                 -1.12620400   -0.87505800   -0.54526700
 H                 -1.12613100    0.90979200   -0.48487200
 H                 -1.12635600   -0.03493300    1.03067100
 Br                 1.73178100    0.00001100   -0.00001600
 Cl                -3.31174600   -0.00000100   -0.00002900

frame.wfn
