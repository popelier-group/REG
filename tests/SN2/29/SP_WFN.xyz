
6

C       -1.320542    0.012707    0.129229
H       -0.913649   -0.536601    0.968098
H       -1.086753   -0.474554   -0.808281
H       -0.980584    1.040047    0.131980
Br        1.850636   -0.017638   -0.181046
Cl       -3.168706    0.030129    0.309966
