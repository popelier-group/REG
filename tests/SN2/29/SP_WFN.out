 Entering Gaussian System, Link 0=/opt/apps/apps/binapps/gaussian/g16c01_em64t_haswell/g16/g16
 Initial command:
 /opt/apps/apps/binapps/gaussian/g16c01_em64t_haswell/g16/l1.exe "/scratch/v69787ff/gau_temp_1513317.29/Gau-175478.inp" -scrdir="/scratch/v69787ff/gau_temp_1513317.29/"
 Default is to use a total of   2 processors:
                                2 via shared-memory
                                1 via Linda
 Entering Link 1 = /opt/apps/apps/binapps/gaussian/g16c01_em64t_haswell/g16/l1.exe PID=    175479.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                11-Nov-2020 
 ******************************************
 --------------------------------
 # b3lyp/6-311++g(d,p) output=wfn
 --------------------------------
 1/38=1,172=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=4,6=6,7=1111,11=2,25=1,30=1,74=-5/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,6=100,9=1/99;
 -------------------
 Single Point Energy
 -------------------
 Symbolic Z-matrix:
 Charge = -1 Multiplicity = 1
 C                    -1.45468  -0.00005   0.00014 
 H                    -1.1262   -0.87506  -0.54527 
 H                    -1.12613   0.90979  -0.48487 
 H                    -1.12636  -0.03493   1.03067 
 Br                    1.73178   0.00001  -0.00002 
 Cl                   -3.31175  0.        -0.00003 
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -1.454684   -0.000053    0.000142
      2          1           0       -1.126204   -0.875058   -0.545267
      3          1           0       -1.126131    0.909792   -0.484872
      4          1           0       -1.126356   -0.034933    1.030671
      5         35           0        1.731781    0.000011   -0.000016
      6         17           0       -3.311746   -0.000001   -0.000029
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.082129   0.000000
     3  H    1.082129   1.785872   0.000000
     4  H    1.082130   1.785887   1.785882   0.000000
     5  Br   3.186465   3.038276   3.038165   3.038500   0.000000
     6  Cl   1.857062   2.416527   2.416549   2.416504   5.043527
                    6
     6  Cl   0.000000
 This structure is nearly, but not quite of a higher symmetry.
 Consider Symm=Loose if the higher symmetry is desired.
 Stoichiometry    CH3BrCl(1-)
 Framework group  C1[X(CH3BrCl)]
 Deg. of freedom    12
 Full point group                 C1      NOp   1
 Largest Abelian subgroup         C1      NOp   1
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -1.320542    0.012707    0.129229
      2          1           0       -0.913649   -0.536601    0.968098
      3          1           0       -1.086753   -0.474554   -0.808281
      4          1           0       -0.980584    1.040047    0.131980
      5         35           0        1.850636   -0.017638   -0.181046
      6         17           0       -3.168706    0.030129    0.309966
 ---------------------------------------------------------------------
 Rotational constants (GHZ):         157.2270933           0.7762604           0.7762603
 Standard basis: 6-311++G(d,p) (5D, 7F)
 There are   126 symmetry adapted cartesian basis functions of A   symmetry.
 There are   121 symmetry adapted basis functions of A   symmetry.
   121 basis functions,   210 primitive gaussians,   126 cartesian basis functions
    31 alpha electrons       31 beta electrons
       nuclear repulsion energy       165.5156463648 Hartrees.
 NAtoms=    6 NActive=    6 NUniq=    6 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=   121 RedAO= T EigKep=  2.73D-04  NBF=   121
 NBsUse=   121 1.00D-06 EigRej= -1.00D+00 NBFU=   121
 ExpMin= 3.50D-02 ExpMax= 4.40D+05 ExpMxC= 1.51D+04 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor=  402 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor=  402 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=       500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Keep R1 ints in memory in canonical form, NReq=29147525.
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 SCF Done:  E(RB3LYP) =  -3074.40274626     A.U. after   11 cycles
            NFock= 11  Conv=0.79D-08     -V/T= 2.0014

 **********************************************************************

            Population analysis using the SCF Density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A)
       Virtual   (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A)
 The electronic state is 1-A.
 Alpha  occ. eigenvalues -- -482.55463-101.41266 -62.20763 -56.02544 -56.02536
 Alpha  occ. eigenvalues --  -56.02536 -10.08556  -9.32689  -8.41632  -7.09066
 Alpha  occ. eigenvalues --   -7.08210  -7.08210  -6.23939  -6.23904  -6.23904
 Alpha  occ. eigenvalues --   -2.35176  -2.35165  -2.35165  -2.35138  -2.35138
 Alpha  occ. eigenvalues --   -0.71211  -0.54494  -0.48782  -0.30932  -0.30931
 Alpha  occ. eigenvalues --   -0.27480  -0.17793  -0.17793  -0.05329  -0.05196
 Alpha  occ. eigenvalues --   -0.05196
 Alpha virt. eigenvalues --    0.10849   0.12163   0.14754   0.14754   0.17457
 Alpha virt. eigenvalues --    0.18196   0.19514   0.19514   0.20647   0.20648
 Alpha virt. eigenvalues --    0.21634   0.22345   0.22345   0.24766   0.28129
 Alpha virt. eigenvalues --    0.30565   0.30565   0.36462   0.40342   0.53345
 Alpha virt. eigenvalues --    0.53346   0.55177   0.58433   0.58434   0.60967
 Alpha virt. eigenvalues --    0.62064   0.62064   0.64125   0.76334   0.76334
 Alpha virt. eigenvalues --    0.79468   0.84280   0.87964   0.87964   0.89248
 Alpha virt. eigenvalues --    0.89250   0.94918   0.96478   0.96478   1.00823
 Alpha virt. eigenvalues --    1.00823   1.02869   1.07029   1.26022   1.26022
 Alpha virt. eigenvalues --    1.26545   1.56760   1.67406   1.67406   1.84711
 Alpha virt. eigenvalues --    1.84711   1.86572   2.09951   2.21572   2.21575
 Alpha virt. eigenvalues --    2.27105   2.27105   2.28324   2.45406   2.45406
 Alpha virt. eigenvalues --    2.55912   2.59085   2.63861   2.63862   2.66694
 Alpha virt. eigenvalues --    2.66694   3.09541   3.28842   3.86883   3.86884
 Alpha virt. eigenvalues --    4.39687   4.39690   4.40365   4.40367   4.50360
 Alpha virt. eigenvalues --    6.75973   7.80386   7.80387   7.85419   9.83815
 Alpha virt. eigenvalues --   24.02927  25.98450  25.98450  26.48655  48.07389
 Alpha virt. eigenvalues --  215.82383 290.01213 290.01213 290.058341020.83049
          Condensed to atoms (all electrons):
               1          2          3          4          5          6
     1  C    5.630753   0.401863   0.401861   0.401869  -0.051824  -0.134011
     2  H    0.401863   0.518678  -0.038012  -0.038013   0.002252  -0.039372
     3  H    0.401861  -0.038012   0.518660  -0.038013   0.002251  -0.039361
     4  H    0.401869  -0.038013  -0.038013   0.518709   0.002255  -0.039393
     5  Br  -0.051824   0.002252   0.002251   0.002255  35.921966   0.050153
     6  Cl  -0.134011  -0.039372  -0.039361  -0.039393   0.050153  17.202229
 Mulliken charges:
               1
     1  C   -0.650510
     2  H    0.192605
     3  H    0.192614
     4  H    0.192588
     5  Br  -0.927052
     6  Cl  -0.000244
 Sum of Mulliken charges =  -1.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C   -0.072704
     5  Br  -0.927052
     6  Cl  -0.000244
 Electronic spatial extent (au):  <R**2>=           1230.2046
 Charge=             -1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=             -4.6381    Y=              0.0444    Z=              0.4538  Tot=              4.6605
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -75.9069   YY=            -44.4635   ZZ=            -44.7616
   XY=              0.2991   XZ=              3.0761   YZ=             -0.0292
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -20.8629   YY=             10.5805   ZZ=             10.2824
   XY=              0.2991   XZ=              3.0761   YZ=             -0.0292
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=             44.7930  YYY=              0.6177  ZZZ=             -2.6182  XYY=              8.5840
  XXY=             -0.2708  XXZ=             -2.7082  XZZ=              8.7318  YZZ=             -0.9375
  YYZ=             -0.7491  XYZ=             -0.0649
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=          -1635.8619 YYYY=            -76.5491 ZZZZ=            -89.9782 XXXY=              8.1647
 XXXZ=             83.6471 YYYX=              5.9320 YYYZ=             -0.5778 ZZZX=             69.1253
 ZZZY=             -0.8841 XXYY=           -260.3565 XXZZ=           -263.9499 YYZZ=            -27.7516
 XXYZ=             -0.2109 YYXZ=             22.8915 ZZXY=              3.0040
 N-N= 1.655156463648D+02 E-N=-7.684341040387D+03  KE= 3.070023630819D+03
 Unable to Open any file for archive entry.
 1\1\GINC-NODE719\SP\RB3LYP\6-311++G(d,p)\C1H3Br1Cl1(1-)\ZZALSPA2\11-No
 v-2020\0\\# b3lyp/6-311++g(d,p) output=wfn\\Single Point Energy\\-1,1\
 C,0,-1.454684,-0.000053,0.000142\H,0,-1.126204,-0.875058,-0.545267\H,0
 ,-1.126131,0.909792,-0.484872\H,0,-1.126356,-0.034933,1.030671\Br,0,1.
 731781,0.000011,-0.000016\Cl,0,-3.311746,-0.000001,-0.000029\\Version=
 ES64L-G16RevC.01\State=1-A\HF=-3074.4027463\RMSD=7.883e-09\Dipole=-1.8
 335775,-0.000047,0.0001142\Quadrupole=-15.7369051,7.8684504,7.8684547,
 -0.0000045,-0.0001736,-0.0000097\PG=C01 [X(C1H3Br1Cl1)]\\@
 The archive entry for this job was punched.

 Writing a WFN file to "frame.wfn"


 SIC AS THE CAWSE OF EWERY THING IS, SIC WILBE THE EFFECT.
 -- PROVERBS AND REASONS OF THE YEAR 1585
           AS REPRINTED IN PAISLEY MAGAZINE 1828.
 Job cpu time:       0 days  0 hours  0 minutes 13.8 seconds.
 Elapsed time:       0 days  0 hours  0 minutes  7.3 seconds.
 File lengths (MBytes):  RWF=     10 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 16 at Wed Nov 11 19:00:15 2020.
